src/bloom.cpp

```diff
@@ -25,37 +25,37 @@ CBloomFilter::CBloomFilter(const unsigned int nElements, const double nFPRate, c
      * - nElements * log(fp rate) / ln(2)^2
      * We ignore filter parameters which will create a bloom filter larger than the protocol limits
      */
-    vData(std::min((unsigned int)(-1  / LN2SQUARED * nElements * log(nFPRate)), MAX_BLOOM_FILTER_SIZE * 8) / 8),
+    m_vData(std::min((unsigned int)(-1  / LN2SQUARED * nElements * log(nFPRate)), MAX_BLOOM_FILTER_SIZE * 8) / 8),
     /**
      * The ideal number of hash functions is filter size * ln(2) / number of elements
      * Again, we ignore filter parameters which will create a bloom filter with more hash functions than the protocol limits
      * See https://en.wikipedia.org/wiki/Bloom_filter for an explanation of these formulas
      */
-    isFull(false),
-    isEmpty(true),
-    nHashFuncs(std::min((unsigned int)(vData.size() * 8 / nElements * LN2), MAX_HASH_FUNCS)),
-    nTweak(nTweakIn),
-    nFlags(nFlagsIn)
+    m_isFull(false),
```

Comment:
The code is follow camelCase and you are changing it to underscore_case.  Bad decision.  Follow the current style.

Reply:
The `m_` convention denotes that the variable is a data member of the class. It is an aid for maintainers to avoid creating side effects by mistake. It doesn't contradict with the current style.

Reply:
I understand!  Thanks!